### PR TITLE
Fix footer link circle dividers' screen reader accessibility by adding aria-hidden

### PR DESCRIPTION
**Pull request by hs4man21**
The screen reader VoiceOver is reading each decorative `·` between the footer links as "middle dot." This is annoying and does not benefit the user, so we should hide these dots from the screen reader via `aria-hidden`. With this change, the links are read in order without the interruption of each middle dot.

### CODE REVIEW

File: app/javascript/mastodon/features/ui/components/link_footer.js [hunk 1]
```diff
@@ -60,31 +60,31 @@ class LinkFooter extends React.PureComponent {
...
           )}
-          {' · '}
+          <span aria-hidden>{' · '}</span>
           <Link key='privacy-policy' to='/privacy-policy'><FormattedMessage id='footer.privacy_policy' defaultMessage='Privacy policy' /></Link>
         </p>
 
         <p>
           <strong>Mastodon</strong>:
-          {' '}
+          <span aria-hidden>{' · '}</span>
```
Comment (ykzts) [inline app/javascript/mastodon/features/ui/components/link_footer.js:79]: This line is just white space. Is this change necessary?

Comment (hs4man21) [reply app/javascript/mastodon/features/ui/components/link_footer.js]: Nice catch! Fixed

**Review by ykzts — APPROVED** [review]

**Review by ClearlyClaire — APPROVED** [review]
